infrastructure/modules/k3s/variables.tf

```
@@ -11,8 +11,11 @@ variable "managers" {
       private_ip = string
       username   = string
     })
-    labels      = map(bool)
-    private_key = string
+    labels       = map(bool)
+    private_key  = string
+    bastion_host = string
+    bastion_port = string
+    bastion_user = string
```

```suggestion
    bastion_host = optional(string)
    bastion_port = optional(string)
    bastion_user = optional(string)
```

This avoids making this a required value (currently breaking the Hetzner integration see)

---

oh yeah, excellent point, hadn't factored in the mandatory nature of those, thank you!

---

Don't worry, I've done it many times before and will sure do again